Rover/sailboat.h

```diff
@@ -116,4 +150,16 @@ class Sailboat
     uint32_t tack_clear_ms;         // system time when tack was cleared
     bool tack_assist;               // true if we should use some throttle to assist tack
     UseMotor motor_state;           // current state of motor output
+
+    // in irons state variables. Dont set directly
+    struct {
+        // system time when in_irons started
+        uint32_t start_ms;
+        // in irons state flag.
+        bool state;
+        // value between -1 and 1 to send to steering when in irons
+        float rudder;
+        // target heading in radians when in irons
+        float heading_rad;
+    } _in_irons;
```

Comment:
The pattern in this lib is not to prefix with `_` for class variables.

Reply:
The Ardupilot  style guide states unequivocally that private member variables for new code should be prefixed with an underscore.